VisualizationBase/src/items/VInfoNode.cpp

```diff
@@ -49,7 +52,12 @@ void VInfoNode::initializeForms()
 void VInfoNode::determineChildren()
 {
 	Super::determineChildren();
-	browser_->setContent(node()->infoHtml());
+	if (node()->hasContentChanged())
```

Comment:
So we should try two things, after you make the change I suggested to InfoNode:
- Either remove this condition entirely and see if the update works as expected
- Or replace this condition with `(revision() != node()->revision())`